src/note.jl

```diff
@@ -112,6 +113,11 @@ end
 
 Base.copy(notes::Notes) = Notes([copy(n) for n in notes], notes.tpq)
 
+"""
+    is_octave(note1::Note,note2::Note) -> Bool
```

Comment:
```suggestion
    is_octave(note1::Note, note2::Note)
```
Adding a space in between makes it more readable.

Reply:
I'll change this